app/controllers/lettings_logs_controller.rb

```diff
@@ -113,6 +114,18 @@ def update_logs
 
 private
 
+  def set_session_filters
+    filter_manager.serialize_filters_to_session("lettings_logs")
+  end
+
+  def session_filters
+    filter_manager.session_filters("lettings_logs")
+  end
+
+  def filter_manager
+    FilterManager.new(current_user:, session:, params:)
```

Comment:
All but one of the public methods in this class accept the filter type as the first argument. Did you consider initializing the class with this value?

Reply:
Good point, think I'll do that